### ISSUE TITLE
Allow user to remain logged in over browser restart

### ISSUE THREAD

**Issue by sebbASF** [apache/incubator-ponymail#82]
It would be useful to have an option to allow the user to remain logged in over browser restart.

**Comment by Humbedooh** [comment]
Agreed - this should be an easy PR, just have a check box and if checked, set a cookie expiration date sometime in the future.
